src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/letters/InvalidCharacterCheck.java

```diff
@@ -14,27 +15,40 @@
 public class InvalidCharacterCheck {
 
     public static final String NEW_LINE_ERROR = " is split over 2 lines. Please correct this before "
-            + "generating a letter";
+            + "generating a %s";
     public static final String DOUBLE_SPACE_ERROR = " contains a double space. Please correct this before"
-            + " generating a letter";
+            + " generating a %s";
 
     private InvalidCharacterCheck() {
     }
 
-    public static List<String> checkNamesForInvalidCharacters(CaseData caseData) {
+    public static List<String> checkNamesForInvalidCharacters(CaseData caseData, String type) {
         List<String> errors = new ArrayList<>();
         List<String> nameOfParties = findAllParties(caseData);
         for (String name : nameOfParties) {
             if (!Strings.isNullOrEmpty(name) && name.contains("  ")) {
-                errors.add(name + DOUBLE_SPACE_ERROR);
+                errors.add(name + String.format(DOUBLE_SPACE_ERROR, type));
             }
             if (!Strings.isNullOrEmpty(name) && name.contains("\n")) {
-                errors.add(name + NEW_LINE_ERROR);
+                errors.add(name + String.format(NEW_LINE_ERROR, type));
```

Comment:
If you include %s at the beginning of NEW_LINE_ERROR and DOUBLE_SPACE_ERROR then you can do

`String.format(NEW_LINE_ERROR, name, type)`

Reply:
refactored, done